src/app/service/eco-news/eco-news.service.ts

```diff
@@ -27,4 +29,26 @@ export class EcoNewsService {
       }
     );
   }
+
+  public sortLastThreeNewsChronologically(): any {
```

Comment:
Please change return type to the appropriate

Reply:
fixed